main.go

```diff
@@ -0,0 +1,66 @@
+package main
+
+import (
+	"github.com/Sean-Der/fail2go"
+	"github.com/Strum355/log"
+	"github.com/go-chi/chi"
+
+	"fail2rest/api"
+	"fail2rest/services"
+
+	"fmt"
+	"net/http"
+	"os"
+	"time"
+)
+
+func main() {
+	// Initialise logger
+	log.InitSimpleLogger(&log.Config{})
```

Comment:
pls have this use JSON logger if in production :) recommend spf13/viper for loading and using config. see windlass codebase for example